src/utils/components/BootOrderModal/DeviceTypeIcon.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+
+import { DeviceType } from '@kubevirt-utils/resources/vm/utils/boot-order/bootOrder';
+import { NetworkIcon, VolumeIcon } from '@patternfly/react-icons';
+
+type DeviceTypeIconProps = {
+  type: DeviceType;
+};
+
+export const DeviceTypeIconMapping = {
```

Comment:
why export? maybe add a default value? why as const needed?

Reply:
I can remove the export and add a default value. The Volume Icon probably.
`as const` is useful in typescript as it allows to tried it just like an enum